common/sgx/tcbinfo.h

```diff
@@ -48,6 +48,9 @@ typedef struct _oe_tcb_info_tcb_level
     uint16_t pce_svn;
     oe_tcb_level_status_t status;
 
+    bool comp_svn_out_of_date;
+    bool pce_svn_out_of_date;
```

Comment:
oe_tcb_info_tcb_level_t and oe_parsed_tcb_info_t exactly map to the the response specified in "GET TCB INFO V2" section in [https://api.portal.trustedservices.intel.com/documentation#pcs-tcb-info-v2](https://api.portal.trustedservices.intel.com/documentation#pcs-tcb-info-v2).

comp_svn_out_of_date and pce_svn_out_of_date are not defined in the spec. They do not belong in this struct.